src/ucp/core/ucp_ep.c

```diff
@@ -2450,7 +2450,7 @@ ucp_ep_config_max_short(ucp_context_t *context, uct_iface_attr_t *iface_attr,
     ssize_t cfg_max_short;
 
     if (!(iface_attr->cap.flags & short_flag) ||
-        context->config.progress_wrapper_enabled) {
+        ucp_context_usage_tracker_enabled(context)) {
```

Comment:
Could you please explain why should we disable short protocol in case when `ucp_context_usage_tracker_enabled` is true?

Reply:
Because we cannot count packets which bypass non-fast path flow. (without init ucp_req)